organizations/admin.py

```diff
@@ -78,6 +78,20 @@ def filter_queryset_by_membership(qs, user,
             )
     return qs
 
+def is_manager(user, organization, facility):
```

Comment:
This function is probably not 100% complete but sufficient for the one use case where it is used now. (see below)